sdks/java/extensions/sql/datacatalog/src/main/java/org/apache/beam/sdk/extensions/sql/example/BeamSqlDataCatalogExample.java

```diff
@@ -30,7 +30,7 @@
 import org.apache.beam.sdk.transforms.MapElements;
 import org.apache.beam.sdk.values.Row;
 import org.apache.beam.sdk.values.TypeDescriptor;
-import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Strings;
```

Comment:
What is the reason to have to change the guava imports.? Shouldn't we stick to the vendored version of guava for Beam related code. I found a similar issue with gRPC imports too https://issues.apache.org/jira/browse/BEAM-7954

Reply:
I could see this happen if the calcite implementation requires a guava type to be passed in (which is likely a bad interface in Calcites implementation IMO).

Reply:
Calcite does, indeed, have ImmutableSet on its public API.

Reply:
Yes, calcite internal has guava type need to be passed in.